projects/libs/core-library-typescript/src/common-library/signatures.ts

```diff
@@ -0,0 +1,65 @@
+import * as crypto from "crypto";
+import * as stringify from "json-stable-stringify";
+import * as fs from "fs";
+import * as _ from "lodash";
+
+export interface SignaturesConfig {
+  publicKeysPath?: string;
+  privateKeyPath?: string;
+}
+
+export class Signatures {
```

Comment:
A refactoring proposal:

- KeyStore - stores your keypair + public keys of all other nodes.  exports: getMyPrivateKey(), getMyPublicKey(), getNodePublicKey(nodeId). 
- KeyManager - uses the key store to get private/public keys, exports: signMessage(message), verifyMessage(nodeId, message)

Reply:
A more minor suggestion:  KeyManager sounds better maybe?

Reply:
+1 on @idandash's suggestion. A `Signatures` module is a bit weird.